reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -635,6 +638,28 @@ class StaticHtmlReporter : Reporter {
     }
 }
 
+private fun DIV.licenseLink(license: String) {
+    val softwareHeritageArchive = "https://archive.softwareheritage.org/browse/content/sha1_git:"
+
+    val pathUrl = getLicensePath(license) ?: return +license
+
+    val licenseFile = File(pathUrl.toURI())
+
+    val sha1Git = HashAlgorithm.SHA1_GIT.calculate(licenseFile)
+
+    a(href = softwareHeritageArchive + sha1Git) {
```

Comment:
How about inlining `softwareHeritageArchive` and using string interpolation with `sha1Git`?

Reply:
To me it's not clear how Software Heritage knows which file to show based on this link. Could you please add an explanation or a link to the documentation of Software Heritage somewhere?